public/app/features/provisioning/RepositoryOverview.tsx

```diff
@@ -0,0 +1,189 @@
+import { useMemo } from 'react';
+
+import { Stack, Box, Text, LinkButton, Card, TextLink, InteractiveTable } from '@grafana/ui';
+import { CellProps } from '@grafana/ui';
```

Comment:
```suggestion
import { Stack, Box, Text, LinkButton, Card, TextLink, InteractiveTable, CellProps } from '@grafana/ui';
```
This needs to be in one import.